### ISSUE TITLE
:art: Double array feels weird

### ISSUE THREAD

**Issue by Amoki** [yezyilomo/django-restql#15]
`GET /users/?query=[["id", "username"]]` feels weird.
* `GET /users/?query=[["id", "username"]]` and `GET /users/15/?query=["id", "username"]` must have different queries while behaving the same way.
* It would imply I can do stuff like `[["id", "username"], ["id"]]` if I want different serializations for my first and second objects.
* `GET /users/?query=[["id", "username", {"groups": [[ "id", "name", {"events": [[ "date", "name" ]]}]]}]]` is hard to follow.  `GET /users/?query=["id", "username", {"groups": [ "id", "name", {"events": ["date", "name"] }]}]` would semantically do the same.

Is this a limitation of distfier?

**Comment by yezyilomo** [comment]
I see your point, of course that has something to do with dictfier, It's a way to specify that an object expected is iterable. In fact am working on a way to simplify query format to something like 
`[id, name, events{date, name}]` which is somehow close to GraphQL query format and more simpler, I think it can be accomplished with the help of API schema. But for the mean time I think I can tweak dictfier a little bit to remove that weird double array. Thanks for the input thou.
